src/Azs.Compute.Admin/readme.md

```diff
@@ -55,7 +55,7 @@ metadata:
 
 ### PSD1 metadata changes
 subject-prefix: ''
-module-version: 1.1.0
```

Comment:
version to 1.2.0 for cmdlet addition(minor version update), we typically update the revision (1.1.1) alone for bug fixes, major version for breaking changes (2.1.0)